src/localization/resources.ts

```diff
@@ -1,5 +1,9 @@
 import components from '@/public/locales/en/components.json';
 import notifications from '@/public/locales/en/notifications.json';
+import actionPanel from '@/public/locales/en/panel-actions.json';
+import exoplanetsPanel from '@/public/locales/en/panel-exoplanets.json';
+import flightControlPanel from '@/public/locales/en/panel-flightcontrol.json';
+import geolocationPanel from '@/public/locales/en/panel-geolocation.json';
```

Comment:
```suggestion
import panelActions from '@/public/locales/en/panel-actions.json';
import panelActions from '@/public/locales/en/panel-exoplanets.json';
import panelFflightControl from '@/public/locales/en/panel-flightcontrol.json';
import panelGeolocation from '@/public/locales/en/panel-geolocation.json';
```
? Makes sure we group the panels in the same order here as above

I you don't do this, let me know because then I should update in my PR :)

Reply:
I find it easier to read it as actionPanel rather than panelActions?

Reply:
I'm ok with either - so go wit the way you have it then and I'll update on my side :)